modules/custom/openy_socrates/modules/openy_autocomplete_path/openy_autocomplete_path.info.yml

```diff
@@ -1,6 +1,6 @@
 name: Open Y Autocomplete Path
 type: module
 description: Implements an autocomplete matcher with path in the label.
-core: 8.x
+core_version_requirement: ^8 || ^9
 package: Open Y
 version: 8.x-1.0
```

Comment:
No newline at end of file

Reply:
fixed https://github.com/ymcatwincities/openy/pull/2174/commits/839562bfa412cf71c9862ca9d9ce54b72f92ff06